test/e2e/operator_test.go

```diff
@@ -3424,10 +3424,11 @@ func TestUnsupportedConfigOverride(t *testing.T) {
 	t.Parallel()
 
 	var tests = []struct {
-		name, unsupportedConfigOverride, env string
+		name, unsupportedConfigOverride, env, defaultValue, value string
 	}{
-		{"dynamic-config-manager", "dynamicConfigManager", "ROUTER_HAPROXY_CONFIG_MANAGER"},
-		{"contstats", "contStats", "ROUTER_HAPROXY_CONTSTATS"},
+		{"dynamic-config-manager", "dynamicConfigManager", "ROUTER_HAPROXY_CONFIG_MANAGER", "true", "false"},
+		{"contstats", "contStats", "ROUTER_HAPROXY_CONTSTATS", "", "true"},
+		{"max-dynamic-servers", "maxDynamicServers", "ROUTER_MAX_DYNAMIC_SERVERS", "2", "7"},
```

Comment:
Should `defaultValue` for `ROUTER_MAX_DYNAMIC_SERVERS` be `"1"`?

Reply:
Yes, it should. `ROUTER_HAPROXY_CONFIG_MANAGER` should have `false` default value too. These tests failed in CI:
```
        --- FAIL: TestAll/parallel/TestUnsupportedConfigOverride (306.33s)
            --- FAIL: TestAll/parallel/TestUnsupportedConfigOverride/dynamic-config-manager (111.43s)
            --- PASS: TestAll/parallel/TestUnsupportedConfigOverride/contstats (83.49s)
            --- FAIL: TestAll/parallel/TestUnsupportedConfigOverride/max-dynamic-servers (111.41s)
```

Reply:
Updated.